pages/common/jdeps.md

```diff
@@ -0,0 +1,24 @@
+# jdeps
+
+> Java Class Dependency Analyzer.
+> More information: <https://docs.oracle.com/en/java/javase/18/docs/specs/man/jdeps.html>.
+
+- Analyze the dependencies of a `.jar` or `.class` file:
+
+`jdeps {{path/to/filename.class}}`
+
+- Print a summary of all dependencies:
+
+`jdeps {{path/to/filename.jar}} -summary`
+
+- Print all class-level dependencies:
```

Comment:
```suggestion
- Print all class-level dependencies of a Jar file:
```

Reply:
```suggestion
- Print all class-level dependencies of a `.jar` file:
```

Would mentioning file extension be better or is the current one fine?

Reply:
Feels the same for me.

Reply:
For `.jar` vs `JAR` or `Jar` after running a very quick and simple text search it seems that without the `.` is more common in other pages so I'd lean that way.